src/index.ts

```diff
@@ -176,22 +185,49 @@ export class Replay implements Integration {
       ...recordingOptions,
     };
 
+    const usingDeprecatedReplaysSamplingRate =
+      sessionSampleRate === undefined && replaysSamplingRate !== undefined;
+    const usingDeprecatedCaptureOnlyOnError =
+      errorSampleRate === undefined && captureOnlyOnError !== undefined;
+
     this.options = {
       flushMinDelay,
       flushMaxDelay,
       stickySession,
       initialFlushDelay,
-      captureOnlyOnError,
-      replaysSamplingRate,
+      sessionSampleRate: usingDeprecatedReplaysSamplingRate
+        ? replaysSamplingRate
+        : sessionSampleRate,
+      errorSampleRate: usingDeprecatedCaptureOnlyOnError
+        ? errorSampleRate
+        : errorSampleRate,
```

Comment:
```suggestion
      errorSampleRate: usingDeprecatedCaptureOnlyOnError
        ? 1.0
        : errorSampleRate,
```

I think we should hard-code the value here, the meaning of `captureOnlyOnError: true` is always `errorSampleRate: 1.0`. If the default in the sdk changes for whatever reason (idk why it would), it's still `1.0` here.

Reply:
Oops yeah, good catch.